client/src/executor.js

```diff
@@ -4,7 +4,7 @@ var vm = require('vm');
 
 module.exports = {
   template: function(code, template){
-    var context = template.type === 'jade' ? { jade: require('jade/runtime.js')} : {};
+    var context = template.type === 'jade' ? { jade: require('./renderers/support/jade-runtime.js')} : {};
```

Comment:
so it looks like we are including the `jade-runtime` lib as a static resource right?